connector/src/main/scala/com/basho/riak/spark/rdd/RiakConnector.scala

```diff
@@ -135,4 +135,26 @@ object RiakConnector extends Logging {
     val config = RiakConnectorConf(hosts, minConnections, maxConnections)
     new RiakConnector(config)
   }
+
+  /**
+   * Returns minConnection from the first available Riak node.
+   * Since all Riak Nodes use the same value it should work fine
+   */
+  def getMinConnectionsPerNode(riakClient: RiakClient): Int = {
+    val nodes = riakClient.getRiakCluster.getNodes
+
+    require(nodes.size() > 0, "At least 1 node required to obtain minConnection info")
+    nodes.get(0).getMinConnections
```

Comment:
There is 'head' method in Scala, which allows to get first element of the collection. I think it is more convenient then .get(0)